spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -393,6 +409,35 @@ public static void configureAnnotatedHttpServices(
                 meterIdPrefixFunctionFactory.get(METER_TYPE, bean.getServiceName()));
     }
 
+    private static void configureExampleHeaders(DocServiceBuilder docServiceBuilder,
+                                                Map<String, Collection<ExampleHeader>> docServiceHeaders) {
+        requireNonNull(docServiceBuilder, "docServiceBuilder");
+        requireNonNull(docServiceHeaders, "docServiceHeaders");
+
+        for (Entry<String, Collection<ExampleHeader>> entry : docServiceHeaders.entrySet()) {
+            for (ExampleHeader exampleHeader : entry.getValue()) {
+                configureExampleHeader(docServiceBuilder, entry.getKey(), exampleHeader.getMethodName(),
+                                       exampleHeader.getExampleHeaders());
+            }
+        }
+    }
+
+    private static void configureExampleHeader(DocServiceBuilder docServiceBuilder,
```

Comment:
`configureExampleHeaders()` ? (s)

Reply:
I created that method to remove duplicate code.

However, according to the comments below, 'ExampleHeader' should be changed to 'ExampleHeaders', so the current `configureExampleHeaders()` name should be changed to something else.

Do not need this method?

Reply:
Not sure I understood correctly, but I guess we can leave `configureExampleHeaders` as it is. It's just a caveat of using a plural form as a class name, so I guess we can just live with it?